pkg/util/validation/validation.go

```diff
@@ -226,6 +226,37 @@ func IsHTTPHeaderName(value string) bool {
 	return httpHeaderNameRegexp.MatchString(value)
 }
 
+var namespacedSysctls = map[string]bool{
+		"kernel.msgmax":          true,
+		"kernel.msgmnb":          true,
+		"kernel.msgmni":          true,
+		"kernel.sem":             true,
+		"kernel.shmall":          true,
+		"kernel.shmmax":          true,
+		"kernel.shmmni":          true,
+		"kernel.shm_rmid_forced": true,
+}
+
+var namespacedPrefixes = []string{
+		"net.",
+		"fs.mqueue.",
+}
+
+// IsWhitelistedSysctlOpt checks that a sysctl option is supported by docker
```

Comment:
I would write this:
```
IsWhitelistedSysctlOpt checks that a sysctl option is whitelisted because it is known
to be namespaced by the Linux kernel. Note that passing this test is required, but not
sufficient: the container runtime might have a stricter check and refuse to launch a pod.
```